### ISSUE TITLE
Starting physics simulation in SceneComposer yields Exception.

### ISSUE THREAD

**Issue by neph1** [jMonkeyEngine/sdk#411]
java.lang.AbstractMethodError: Receiver class com.jme3.bullet.BulletAppState does not define or inherit an implementation of the resolved method abstract getId()Ljava/lang/String; of interface com.jme3.app.state.AppState.
	at com.jme3.app.state.AppStateManager.attach(AppStateManager.java:142)
	at com.jme3.gde.core.scene.SceneApplication$13.call(SceneApplication.java:672)
	at com.jme3.app.AppTask.invoke(AppTask.java:147)
	at com.jme3.app.LegacyApplication.runQueuedTasks(LegacyApplication.java:750)
	at com.jme3.app.LegacyApplication.update(LegacyApplication.java:765)
[catch] at com.jme3.gde.core.scene.SceneApplication.update(SceneApplication.java:296)

This was also noticed in the VehicleEditor

**Comment by neph1** [comment]
So there exists a BulletAppState inside engine/jme3-bullet folder, which is not the same being opened if I click  it in SceneApplication. I assume this appstate either overrides, or should override some other instance of it on the classpath. Does anyone know how this should work?

**Comment by MeFisto94** [comment]
Not sure about the classpath problems, but certainly we added getId() in some oldish jme, 3.2 or 3.3, so said jme3-bullet seems to be older. What is it even? isn't it supposed to be provided by Minie in more recent builds? (but then engine/ sounds like you're still manually building, and not updating the engine folder to the relevant jme version).

**Comment by neph1** [comment]
I dug some further, 3.1 it seems, coming from "corelibs". If it's my local build, I'll see if I can update it.

**Comment by MeFisto94** [comment]
https://github.com/jMonkeyEngine/sdk/blob/v3.3/settings.gradle is still using the composite build, however on master that's not the case, so in theory you should be able to drop the engine folder, even.

**Comment by neph1** [comment]
Seems to be a local issue. Unresolved, but I found a work around. For some reason it insists on picking jme3-bullet-3.0.0-beta1 when buildSdk is run, even though it's not present in the build file! I don't dare doing a full clean since I have mixtured with the sdk folder to make it behave like a gradle build in the first place.
TLDR; I was able to make it work by changing build.properties in the build folder, and use a compatible jme3-bullet
